src/Swarm/Game/Scenario/WorldDescription.hs

```diff
@@ -21,23 +21,27 @@ import Witch (into)
 ------------------------------------------------------------
 
 -- | A world palette maps characters to 'Cell' values.
-newtype WorldPalette = WorldPalette
-  {unPalette :: KeyMap Cell}
+newtype WorldPalette c = WorldPalette
+  {unPalette :: KeyMap c}
```

Comment:
Would it be better to have `newtype WorldPalette e = WorldPalette { unPalette :: KeyMap (PCell e) }`?  I am honestly not sure.  Or maybe we should just double down on the generality and call it `Palette` instead of `WorldPalette` --- it maps characters to anything.